### PR TITLE
MTM-49961/MTM-47816 remove esper reference from microservice SDK…

### PR DESCRIPTION
**Pull request by jkot-softwareag**
…real-time notification example

### CODE REVIEW

**Review by skom-sag — APPROVED** [review]

File: content/microservice-sdk/rest-bundle/microservice-development.md [hunk 1]
```diff
@@ -560,16 +554,17 @@ HTTP/1.1 200 OK
 }]
 ```
 
-After the handshake, the client needs to subscribe to the output of the above rule. This is done using a POST request with the module name and the statement name as subscription channel. In our example, we used the module name "myRule" and did not give a name to the `select` statement ("@Name('')"), so the subscription channel is "/myRule/\*".
+After the handshake, the client needs to subscribe to the particular device channel or all devices channel (*). This is done using a POST request with the name of the channel.
```
Comment (rssag) [inline content/microservice-sdk/rest-bundle/microservice-development.md:557]:
```suggestion
After the handshake, the client must subscribe to the particular device channel or all devices channel (*). This is done using a POST request with the name of the channel. 
```

File: content/microservice-sdk/rest-bundle/microservice-development.md [hunk 2]
```diff
@@ -560,16 +554,17 @@ HTTP/1.1 200 OK
 }]
 ```
 
-After the handshake, the client needs to subscribe to the output of the above rule. This is done using a POST request with the module name and the statement name as subscription channel. In our example, we used the module name "myRule" and did not give a name to the `select` statement ("@Name('')"), so the subscription channel is "/myRule/\*".
+After the handshake, the client needs to subscribe to the particular device channel or all devices channel (*). This is done using a POST request with the name of the channel. 
+Let's assume that in our example we want to receive notifications about events from the device with internal id 24800. In this case the subscription channel is "/events/24800".
```
Comment (rssag) [inline content/microservice-sdk/rest-bundle/microservice-development.md:558]:
```suggestion
For example, if you want to receive notifications about events from the device with internal ID "24800", the subscription channel is "/events/24800".
```

**Review by rssag — CHANGES_REQUESTED** [review]

**Review by rssag — APPROVED** [review]

File: content/microservice-sdk/rest-bundle/microservice-development.md (hunk 1 above)
Comment (jkot-softwareag) [reply content/microservice-sdk/rest-bundle/microservice-development.md]: done

File: content/microservice-sdk/rest-bundle/microservice-development.md (hunk 2 above)
Comment (jkot-softwareag) [reply content/microservice-sdk/rest-bundle/microservice-development.md]: done

**Pull request merged**